package.json

```diff
@@ -70,6 +69,8 @@
     "@applitools/eyes-cypress": "^3.9.18",
     "axios-mock-adapter": "^1.17.0",
     "cypress": "^4.2.0",
+    "enzyme": "^3.11.0",
+    "enzyme-to-json": "^3.4.4",
```

Comment:
Not sure why enzyme-to-json was added here: https://www.npmjs.com/package/enzyme-to-json

Reply:
Doesn't look like we're using enzyme-to-json at all, so we might be able to remove it.